wiki/Article_Styling_Criteria/en.md

```diff
@@ -163,19 +209,7 @@ Use bold when you need to caution or note something's importance. When bolding,
 
 #### Italics
 
-Use italics only when naming a video game or the name of a work.
-
----
-
-You must use italics when writing a note, and for main page and see also links. For example:
-
-```
-*Main page: [Welcome](/wiki/Welcome).*
-
-*See also: [Guides](/wiki/Guides).*
-
-*Note: You will need a decent internet connection to connect to Bancho and submit scores.*
-```
+Use italics only when naming a video game or the name of a work. osu!, the game, is exempt from this.
```

Comment:
Should you note that asterisks are preferred than underscores? Also double asterisks for bold.

Reply:
As far as I know, no one ever said or suggested to only use asterisks over underscores.

Reply:
heck with it, alright